src/Relation/Nullary/Negation/Core.agda

```diff
@@ -48,23 +47,26 @@ _¬-⊎_ = [_,_]
 ------------------------------------------------------------------------
 -- Uses of negation
 
+weak-contradiction : .A → ¬ A → Whatever
+weak-contradiction a ¬a = ⊥-elim-irr (¬a a)
```

Comment:
I think the name would be `weakContradiction`?

Reply:
`contradiction-irr` is at least congruent with `⊥-elim-irr` so I'll make that change for the time being